src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/SymbolFactory.vb

```diff
@@ -61,6 +61,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return False
         End Function
 
+        Friend Overrides Function IsAcceptedIsExternalInitModifierType(type As TypeSymbol) As Boolean
+            ' VB doesn't deal with init-only members.
+            Return False
```

Comment:
> Return False [](start = 11, length = 13)

Is this code path reachable? Are we testing it?

Reply:
Will be tested by VB consumption of C#-produced PE. Added PROTOTYPE

---
In reply to: [413381872](https://github.com/dotnet/roslyn/pull/43275#discussion_r413381872) [](ancestors = 413381872)